cli/src/pcluster/config/cluster_config.py

```diff
@@ -2161,9 +2178,11 @@ class SlurmQueue(_CommonQueue):
     def __init__(
         self,
         allocation_strategy: str = None,
+        custom_slurm_settings: Dict = None,
         **kwargs,
     ):
         super().__init__(**kwargs)
+        self.custom_slurm_settings = Resource.init_param(custom_slurm_settings, default={})
```

Comment:
I'm wondering when we need `Resource.init_param()` and when not...

Reply:
Not 100% sure, my understanding is that we use it when we want to have a default value and avoid `None`